app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -74,7 +69,7 @@ def valid_auth_tokens
 
     def analytics_properties
       {
-        rendered_event_count: security_event_tokens.keys.count,
+        rendered_event_count: 1, # FIXME
```

Comment:
See suggestion above on line 49 for context

```suggestion
        rendered_event_count: security_event_tokens.count,
```